build.sh

```diff
@@ -35,7 +35,11 @@ rm -rf vendor/pb
 git clone https://github.com/PitchBlackRecoveryProject/vendor_pb -b pb vendor/pb --depth=1
 
 echo -e "\nGetting the Device Tree on place"
+if [ "${CIRCLE_PROJECT_USERNAME}" == "PitchBlackRecoveryProject" ]; then
 git clone --quiet --progress https://$GitHubName:$GITHUB_TOKEN@github.com/PitchBlackRecoveryProject/${CIRCLE_PROJECT_REPONAME} -b ${CIRCLE_BRANCH} device/${VENDOR}/${CODENAME}
+else
+git clone https://github.com/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME} -b ${CIRCLE_BRANCH} device/${VENDOR}/${CODENAME}
```

Comment:
there is no need of this since we already doing so this at lunch or breakfast.

Reply:
that changes need if someone building other than organization, script cloning tree from org

Reply:
Hmm.
This script will be more _user-friendly_ if this is implemented.
Any user or _wanna-be developer_ can test their build in their Personal CI Plan.